src/main/java/com/spider/solomon/order/CustomerOrderDao.java

```diff
@@ -34,7 +36,13 @@ public CustomerOrder getOrderById (int orderId) {
 	 */
 	public List<CustomerOrder> getOrdersByCustomerId (int customerId) {
 		String sql = "SELECT * FROM customer_orders WHERE customer_id = ?";
-		return jdbcTemplate.query(sql, ORDER_MAPPER, customerId);
+
+		List<CustomerOrder> ordrs = jdbcTemplate.query(sql, ORDER_MAPPER, customerId);
+		ordrs.forEach(order -> {
+			order.setItems(orderItemDao.getOrderItemsByOrderId(order.getOrderId()));
```

Comment:
I think that for customers with a lot of orders, this may become a performance issue. Should we consider doing a join in the sql instead, so it's just one query? Or if not one query, then two, with `orderItemDao.getOrderItemsByCustomerId(..)` instead of `orderItemDao.getOrderItemsByOrderId(..)`?